### PR TITLE
Add outline of AtoM DIP Upload process

### PR DESCRIPTION
**Pull request by djjuhasz**

### CODE REVIEW

File: admin-manual/installation/dashboard-config.rst [hunk 1]
```diff
@@ -261,9 +261,26 @@ AtoM DIP upload
...
+   and AtoM share a common filesystem (e.g. a shared network directory) this 
+   step is unnecessary.
+
+#. Archivematica sends a REST request to AtoM to tell AtoM that which archival
+   description is the target of the DIP. The DIP target description is 
+   identified by the description's "slug".  The actual upload of the DIP 
+   contents to AtoM is done via a background job, and may take some time to
+   process if a large DIP is uploaded.
+
+#. An AtoM background worker uploads the DIP metadata (METs file) and digital
```
Comment (sromkey) [inline admin-manual/installation/dashboard-config.rst]: Nitpick- METS instead of METs.

**Review by sromkey — COMMENTED** [review]
Other than small nitpick looks good for style, etc! I'll let @sevein comment r.e. technical detail.

**Review by sevein — APPROVED** [review]
This is great, thanks!

**Comment by sevein** [comment]
23c5ac7ca1ff1455333c369c764d7a8a999de377

**Pull request closed without merging**